package.json

```diff
@@ -26,27 +26,29 @@
   },
   "homepage": "https://github.com/ProjectWyvern/wyvern-js#readme",
   "dependencies": {
-    "0x.js": "^0.29.2",
-    "@0xproject/abi-gen": "^0.1.1",
-    "@0xproject/assert": "^0.0.11",
-    "@0xproject/json-schemas": "^0.7.3",
-    "@0xproject/utils": "^0.1.3",
-    "@0xproject/web3-wrapper": "^0.1.4",
+    "@0x/abi-gen": "^5.7.0",
+    "@0x/assert": "^3.0.30",
+    "@0x/base-contract": "^6.4.4",
+    "@0x/json-schemas": "^6.4.0",
+    "@0x/utils": "^6.4.4",
+    "@0x/web3-wrapper": "^7.6.1",
     "bn.js": "^4.11.8",
     "ethereumjs-abi": "https://github.com/ProjectWyvern/ethereumjs-abi.git",
     "ethereumjs-util": "^5.1.3",
+    "ethers": "^4.0.49",
```

Comment:
Why is `ethers` needed?

Reply:
the generated `abi` files use `ethers`. when running `yarn abi_gen`. not sure if there's a better way to convey that it's due to a build artifacts needing it.